bindings/objective-c/librpc/librpc.m

```diff
@@ -34,114 +34,123 @@
 #include <rpc/typing.h>
 
 #pragma mark - RPCObject
+@interface RPCObject ()
+@property (nonatomic, unsafe_unretained) rpc_object_t obj;
```

Comment:
Can you explain that change? How is a private property different than member field? Does it behave differently than the member field removed blow?

Reply:
The difference is subtle, but the idea is basically that using properties instead of member ivars is preferred for debugging because it is possible to set a breakpoint when accessing or mutation, they can be overridden to add logging or other test tasks and provide well understood and properly managed memory allocation and deallocation in the automated setter/getter methods. Though there is some debate on the subject, using properties over member variables is generally considered a best practice in cocoa.

Reply:
OK, then it should at least be read-only.